src/libraries/System.Private.CoreLib/src/System/IO/FileStream.cs

```diff
@@ -49,6 +49,11 @@ public FileStream(IntPtr handle, FileAccess access, bool ownsHandle, int bufferS
                 ValidateHandle(safeHandle, access, bufferSize, isAsync);
 
                 _strategy = FileStreamHelpers.ChooseStrategy(this, safeHandle, access, bufferSize, isAsync);
+
+                if (!_strategy.RequiresFinalizer)
+                {
+                    GC.SuppressFinalize(this);
+                }
```

Comment:
Do we really need to do this?  Do we need to be bending over backwards to try to improve the performance of code that neglects to dispose of a FileStream?

Reply:
we don't, I am going to revert this change